### PR TITLE
[SMALLFIX] use provided jline in spark profile

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -689,6 +689,11 @@
         <version>3.0.1</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
```
Comment (apc999) [inline pom.xml]: wrong location. we should put it earlier int he session where this is the "Provided scope" section

Comment (peisun1115) [inline pom.xml]: done

**Comment by peisun1115** [comment]
@apc999 Any more comments?

File: pom.xml [hunk 2]
```diff
@@ -309,6 +309,11 @@
         <version>${metrics.version}</version>
       </dependency>
       <dependency>
```
Comment (jsimsa) [inline pom.xml]: keep alphabetical order (this should come after `javax.servlet` and before `log4j`)

Comment (peisun1115) [inline pom.xml]: done

**Pull request merged**